scaffolding-chef-inspec/lib/scaffolding.sh

```diff
@@ -60,9 +60,17 @@ do_default_before() {
       build_line "$message"
       exit 1
     else
-      inspec compliance login $scaffold_automate_server_url \
+      if [ ! $scaffold_compliance_insecure ]; then
```

Comment:
I think this looks good is. Do you also need the insecure for the archive command or is that working cause the compliance login was insecure?